Firestore/Source/Core/FSTView.mm

```diff
@@ -216,8 +216,8 @@ - (FSTViewDocumentChanges *)computeChangesWithDocuments:(FSTMaybeDocumentDiction
       newDoc = (FSTDocument *)maybeNewDoc;
     }
     if (newDoc) {
-      FSTAssert([key isEqual:newDoc.key], @"Mismatching key in document changes: %@ != %@", key,
-                newDoc.key);
+      FSTAssert([key isEqual:newDoc.key], @"Mismatching key in document changes: %@ != %s", key,
+                newDoc.key.path().CanonicalString().c_str());
```

Comment:
Should we add a ToString() on DocumentKey to cut down on this clutter in the log lines?

Reply:
done